src/main/java/chess/controller/ChessController.java

```diff
@@ -1,26 +1,104 @@
 package chess.controller;
 
+import chess.dao.*;
+import chess.dao.converter.PieceConverter;
 import chess.domain.Board;
 import chess.domain.BoardFactory;
 import chess.domain.ChessGame;
-import chess.domain.Positions;
+import chess.domain.piece.Piece;
+import chess.domain.piece.character.Team;
+import chess.domain.position.Position;
+import chess.domain.position.Positions;
 import chess.dto.BoardDto;
+import chess.view.Command;
 import chess.view.InputView;
 import chess.view.OutputView;
 
+import java.sql.Connection;
+import java.util.HashMap;
+import java.util.Map;
+
 public class ChessController {
+    private Connection connection;
+
+    public ChessController() {
+        connection = ConnectionGenerator.getConnection("chess");
+        new SettingDao(connection).settingTable();
+        new PieceDao(connection).settingPiece();
+        ConnectionGenerator.closeConnection(connection);
+    }
+
     public void play() {
-        InputView.inputStartCommand();
-        Board board = new Board(BoardFactory.generateStartBoard());
-        ChessGame chessGame = new ChessGame(board);
+        connection = ConnectionGenerator.getConnection("chess");
+        String gameId = InputView.inputGameId();
+        ChessGame chessGame = makeGame(gameId);
+        OutputView.printChessBoard(new BoardDto(chessGame.board()));
+        Command command;
+        while (!chessGame.findCheck().isCheckMate() && (command = InputView.inputCommand()) != Command.END) {
+            gamePlayByCommand(chessGame, command, gameId);
+        }
+        printChessScore(chessGame);
+        ConnectionGenerator.closeConnection(connection);
+    }
 
-        OutputView.printChessBoard(new BoardDto(board));
+    private ChessGame makeGame(String gameId) {
+        ChessGameDao chessGameDao = new ChessGameDao(connection);
+        boolean isPlayed = chessGameDao.checkById(gameId);
+        if (!isPlayed) {
+            chessGameDao.addChessGame(gameId);
+        }
+        return new ChessGame(makeBoard(gameId, isPlayed), Team.WHITE);
```

Comment:
의견이 궁금해요!

new ChessGame(makeBoard(gameId, isPlayed)); 
와 같이 첫 생성 시 Team.WHITE를 생성자 내부에서 지정해주는 flow는 어때요?

Reply:
원래 코드는 White로 생성자 내부에서 지정하도록 하였습니다!
최대한 변경에 유연한 코드로 하면 좋지 않을까? 하는 생각에 외부에서 첫 시작 Team을 주입하도록 수정하였습니다!
그리고 게임을 로드할 때, 게임의 현재 누구의 turn인지까지 로드해야하므로 team을 주입받을 수 있어야한다고 생각해서 수정하였습니다!😁

지금 코드는 White로 고정되어있지만,, 사실 다음과 같은 코드여야합니다! 제가 실수를 했네요 !
```suggestion
       return new ChessGame(makeBoard(gameId, isPlayed), chessGameDao.findTeamById(gameId));
```

Reply:
ㅋㅋ아하!! 저도 그건 미처 발견을 못했네요 👍